### PR TITLE
Update rex related packages [RPM]

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

**Comment by domcleal** [comment]
[test]

File: rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec [hunk 1]
```diff
@@ -30,15 +30,15 @@ Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.7.11
 Requires: %{?scl_prefix}rubygem(foreman-tasks) < 0.8.0
```
Comment (domcleal) [inline rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: Runtime requirement must also be updated for ~> 0.8.0.

Comment (iNecas) [inline rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec]: Fixed, thanks for noticing!

**Comment by domcleal** [comment]
Merged and building, thanks @iNecas.

**Pull request merged**